Packs/FileOrbis/README.md

```diff
@@ -0,0 +1,9 @@
+FileOrbis content pack enables you to manage users.
+
+## What does this pack do?
+
+The integration in this pack enables you to:
+- Change of the user status
```

Comment:
What else does it do? Why would a user want to do that? Can you explain the use case better?

Reply:
Explanation in this file updated